svc-bip-api/src/test/java/gov/va/vro/bip/service/RMQIntegrationTest.java

```diff
@@ -0,0 +1,193 @@
+package gov.va.vro.bip.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import gov.va.vro.bip.model.BipClaim;
+import gov.va.vro.bip.model.BipUpdateClaimResp;
+import io.jsonwebtoken.Claims;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.web.client.RestTemplate;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+
+/**
+ * Does same thing as BipApiServiceTest but through RMQ instance. Assumes RMQ broker is available
+ * locally.
+ */
+@Disabled("needs an RMQ broker, which is not available in github build env.")
```

Comment:
There are ways to still let these tests run. There is spring [support](https://docs.spring.io/spring-amqp/docs/current/reference/html/testing.html) for testing with rabbitmq. But i would recommend using [test-containers](https://gaddings.io/testing-spring-boot-apps-with-rabbitmq-using-testcontainers/).

Reply:
I'm sure you're right.  And I'll get this done.  But tracked by story #1777